pyanaconda/modules/common/structures/payload.py

```diff
@@ -0,0 +1,60 @@
+#
+# DBus structures for the payload data.
+#
+# Copyright (C) 2020 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+
+from dasbus.structure import DBusData
+from dasbus.typing import *  # pylint: disable=wildcard-import
+from pyanaconda.core.constants import URL_SOURCE_BASE
+
+
+class URLData(DBusData):
+    """Structure which contains link to URL and type of URL."""
+
+    def __init__(self):
+        self._url = ""
+        self._url_type = URL_SOURCE_BASE
+
+    @property
+    def url(self) -> Str:
+        """Get link to the repository."""
+        return self._url
+
+    @url.setter
+    def url(self, url: Str):
+        """Set link to the repository."""
+        self._url = url
+
+    @property
+    def url_type(self) -> Str:
```

Comment:
This property could be called just `type`.

Reply:
Fixed.